project1-sps/script.js

```diff
@@ -0,0 +1,155 @@
+//Objective SPS human vs computer
+// What is the input going to be? scissors, paper, stone
+// What should the output be?
+//scissors beats paper,
+//paper beats stone, and
+//stone beats scissors
+//same will be draw
+// How will the computer's option be randomly generated? math.random
+// How many different cases are there?
+// Scissors Paper Stone
+// 0        0     0 (not possible)
+// 0        0     1 (YES, BOTH STONE)
+// 0        1     0 (YES, BOTH PAPER)
+// 0        1     1 (YES)
+// 1        0     0 (YES, BOTH SCISSORS)
+// 1        0     1 (YES)
+// 1        1     0 (YES)
+// 1        1     1 (not possible)
+
+// function to trigger button click on enter key
+document
+  .getElementById("input-field")
+  .addEventListener("keyup", function (event) {
+    event.preventDefault();
+    if (event.keyCode === 13) {
+      document.getElementById("submit-button").click();
+    }
+  });
```

Comment:
this is impressive. good job on going beyond the material taught so far and googling how to do this. arguably thats the most important takeaway from this course.

also, this style of interacting with HTML elements in JS is called DOM manipulation and we will be covering it briefly on day 8 so you can look forward to it